libsolidity/analysis/NameAndTypeResolver.cpp

```diff
@@ -47,7 +47,9 @@ NameAndTypeResolver::NameAndTypeResolver(
 	if (!m_scopes[nullptr])
 		m_scopes[nullptr].reset(new DeclarationContainer());
 	for (Declaration const* declaration: _globals)
-		m_scopes[nullptr]->registerDeclaration(*declaration);
+	{
```

Comment:
Some compilers, most notably MSVC have problems with macros if they are the single statement in a loop body. Because of that, I always put braces around macros in such cases.

Reply:
Ah, OK, that makes sense.